pkg/services/pluginsintegration/pluginsintegration.go

```diff
@@ -0,0 +1,30 @@
+package pluginsintegration
+
+import (
+	"github.com/google/wire"
+	"github.com/grafana/grafana/pkg/plugins"
+	"github.com/grafana/grafana/pkg/plugins/config"
+	"github.com/grafana/grafana/pkg/plugins/manager/client"
+	"github.com/grafana/grafana/pkg/plugins/manager/registry"
+	"github.com/grafana/grafana/pkg/services/oauthtoken"
+	"github.com/grafana/grafana/pkg/services/pluginsintegration/clientmiddleware"
+	"github.com/grafana/grafana/pkg/setting"
+)
+
+// WireSet provides a wire.ProviderSet of plugin providers.
+var WireSet = wire.NewSet(
```

Comment:
Not sure it make sense to expose a new `wire.ProviderSet` since we don't do this anywhere else. The alternative would be to put the providers directly in wire.go as we normally do.

The bigger story here is that this `wire.ProviderSet` could provide all providers related to plugin management rather than keeping them in wire.go